h2o-r/h2o-package/R/xgboost.R

```diff
@@ -106,7 +107,7 @@ h2o.xgboost <- function(x, y, training_frame,
                         offset_column = NULL,
                         weights_column = NULL,
                         stopping_rounds = 0,
-                        stopping_metric = c("AUTO", "deviance", "logloss", "MSE", "RMSE", "MAE", "RMSLE", "AUC", "lift_top_group", "misclassification", "mean_per_class_error", "custom", "custom_increasing"),
+                        stopping_metric = c("AUTO", "deviance", "logloss", "MSE", "RMSE", "MAE", "RMSLE", "AUC", "lift_top_group", "misclassification", "mean_per_class_error", "anomaly_score", "custom", "custom_increasing"),
```

Comment:
Same point as Veronica has pointed out.  Please fix the gen_python.py gen_R.py to get rid of anomaly_score for all other algos except isolation forest.